batch2/batch/worker.py

```diff
@@ -611,7 +635,7 @@ def __init__(self):
         if job is None:
             raise web.HTTPNotFound()
 
-        asyncio.ensure_future(job.delete())
+        job.delete()
```

Comment:
This will at least need an await.  Curious, why this change?

Reply:
I didn't mean to change it. I didn't look at the diff carefully enough. Sorry about that.